integreat_cms/cms/forms/regions/region_form.py

```diff
@@ -424,6 +426,27 @@ def clean_hix_enabled(self):
             )
         return cleaned_hix_enabled
 
+    def clean_deepl_usage_remaining(self):
+        """
+        Validate the deepl_usage_remaining field (see :ref:`overriding-modelform-clean-method`).
+
+        :return: The validated field
+        :rtype: bool
+        """
+        cleaned_deepl_usage_limit = self.cleaned_data.get("deepl_usage_limit")
+        cleaned_deepl_usage_remaining = self.cleaned_data.get("deepl_usage_remaining")
+        if cleaned_deepl_usage_limit is None and cleaned_deepl_usage_remaining is None:
+            return self.cleaned_data.get("deepl_usage_remaining")
+        if cleaned_deepl_usage_remaining > cleaned_deepl_usage_limit:
+            self.add_error(
+                "deepl_usage_remaining",
+                _(
+                    "The remaining number of words can not be larger than the total number of allowed words."
+                ),
+            )
+            return False
+        return self.cleaned_data.get("deepl_usage_remaining")
+
```

Comment:
```suggestion
```
As @osmers suggested, I'd make this a readonly field - and then, we also don't need to validate the input.